### DISCUSSION TITLE
Does this project also predict how much money a user should bet to maximize his profits?

### DISCUSSION THREAD

**Discussion by Hemang999** [kyleskom/NBA-Machine-Learning-Sports-Betting#348] [General]
Hi,

Does this project also predict how much money a user should bet to maximize his profits? And also generate a list of the most profitable bets?

**Comment by Atrone** [comment]


**Reply by STRATZ-Ken** [reply]
He is scamming people. Dont email him.

**Comment by Atrone** [comment]


**Reply by Emillzz45** [reply]
You're a literal cuck who showed a handful of bet slips and a notepad on a phone. You came in here bragging how much money you make trading and with this.. again, it's rather weird you are so active here advertising something multiple people have said provide little value.

**Comment by Atrone** [comment]


**Reply by Emillzz45** [reply]
Go look in the mirror bud

**Comment by Atrone** [comment]


**Reply by Emillzz45** [reply]
Everyone knows the type of person you are. Don’t need to expose yourself as a loser even more.

**Comment by STRATZ-Ken** [comment]
@kyleskom I ask you again. please ban this guy.

**Comment by Emillzz45** [comment]
@kyleskom For-real. An absolute clown to this project/community.

**Comment by Emillzz45** [comment]
You have provided 9 bets-lips from 2022-23 and a notepad.. if you haven't noticed everyone stopped responding then. 💩
